StortSpelProjekt/Game/src/Gamefiles/Components/RangeComponent.cpp

```diff
@@ -91,6 +92,11 @@ float component::RangeComponent::GetAttackInterval() const
 	return m_AttackInterval;
 }
 
+void component::RangeComponent::MakeProjectileBouncy(bool bouncy)
+{
+	m_ProjectileRestitution = 0.5f * static_cast<float>(bouncy);
```

Comment:
Should this scale with level of the upgrade? Or is the upgrade only supposed to be bought once?

Reply:
It's only supposed to be bought once. Hence the max level 1. Could possibly make a longer time to live for higher levels. Would be easy to implement in the future if we feel it is needed